pravega/pravega
# Issue 5585: Pravega Installation on Kubernetes Update

Signed-off-by: dellThejas <Thejas.Vidyasagar@dell.com>

**Change log description**  
Updated versions and steps required in documentation for installing Pravega on Kubernetes

**Purpose of the change**  
Fixes #5585 

**What the code does**  
Modified Kubernetes version, and bookkeeper version, and updated Pravega configuration file required for installation on Kubernetes.

**How to verify it**  
Following the updated docs to setup Pravega on Kubernetes will ensure the changes.
